### DISCUSSION TITLE
Can I turn off image optimization?

### DISCUSSION THREAD

**Discussion by richardsdev** [parcel-bundler/parcel#7201] [Q&A] [answered]
Hi. It says in the docs, that Parcel optimizes images by default during production builds. I am planning on using something else which already does that, so would rather turn that off. Is there a configuration option for that? I couldn't see it mentioned in the docs. Any help would be greatly appreciated.

**Comment by mischnic** [comment, marked as answer]
Use this .parcelrc:
```json
{
  "extends": "@parcel/config-default",
  "optimizers": {
    "*.{jpg,jpeg,png}": []
  }
}
```
(But this is a lossless re"packing" of these images, so I don't think there is a reason to turn this off)

**Reply by richardsdev** [reply]
Thanks so much for the swift reply! The reason I will be turning it off is that I am using a plugin for 11ty which does responsive images, this creates multiple copies of images and generates the HTML picture element in my output. The images will be processed to already be as optimized as possible, so Parcel would be trying to optimize that further, but am assuming it would only lead to degradation and extra build time.
